typescript/infra/config/environments/mainnet/core/rust/celo_config.json

```diff
@@ -76,7 +76,7 @@
       "domain": "6648936",
       "name": "ethereum",
       "rpcStyle": "ethereum",
-      "finalityBlocks": "15",
+      "finalityBlocks": "20",
```

Comment:
Looks like the configs that are actually used by the agents in `rust/config` were using 20 but the `typescript/infra` ones were not updated. I just filed https://github.com/abacus-network/abacus-monorepo/issues/779 so we just get rid of the unnecessary configs in `typescript/infra`